protos/flyteidl/core/workflow.proto

```diff
@@ -194,3 +199,15 @@ message WorkflowTemplate {
     // workflow defaults
     WorkflowMetadataDefaults metadata_defaults = 7;
 }
+
+// Optional task node overrides that will be applied at task execution time.
+message TaskNodeOverrides {
+    // A customizable interface to convey resources requested for a task container. 
+    Resources resources = 1;
+
+    // Environment variables will be set as the task container is starting up.
+    repeated KeyValuePair env = 2;
```

Comment:
what about config?

Reply:
You mean also add config here? Or use config to store env bars?